webrender/src/mask_cache.rs

```diff
@@ -95,15 +97,14 @@ impl MaskCacheInfo {
     pub fn update(&mut self,
                   source: &PrimitiveClipSource,
                   transform: &Matrix4D<f32>,
-                  clip_rect: &Rect<f32>,
                   clip_store: &mut GpuStore<GpuBlock32>,
                   device_pixel_ratio: f32,
                   aux_lists: &AuxiliaryLists) {
 
         if self.local_rect.is_none() {
-            let mut local_rect = Some(clip_rect.clone());
+            let mut local_rect = Some(rect_from_points_f(-MAX_COORD, -MAX_COORD, MAX_COORD, MAX_COORD));
```

Comment:
oh, I was thinking if we have an omni-covering rectangle on multiple occasions :)